cassandra-schema.cql

```diff
@@ -1511,7 +1511,7 @@ CREATE TABLE brig_test.oauth_auth_code (
     redirect_uri blob,
     scope set<text>,
     user uuid
-) WITH bloom_filter_fp_chance = 0.1
+) WITH bloom_filter_fp_chance = 0.01
```

Comment:
Do you know where this comes from? Have we changed anything in how we generate this file?